tests/test_spectrum_document.py

```diff
@@ -11,7 +10,7 @@ def test_spectrum_document_init_n_decimals_default_value_no_losses():
     intensities = np.array([0, 0.01, 0.1, 1], dtype="float")
     metadata = dict(precursor_mz=100.0)
     spectrum = Spectrum(mz=mz, intensities=intensities, metadata=metadata)
-    spectrum_document = SpectrumDocument(spectrum)
+    spectrum_document = SpectrumDocument(spectrum, loss_mz_from = 0.0, loss_mz_to = -1.0)
```

Comment:
Since you have split SpectrumDocument and SpectrumDocumentWithLosses, the loss_mz_from and loss_mz_to should be removed here. (it should also fail in the tests)